llvm/lib/Transforms/InstCombine/InstructionCombining.cpp

```diff
@@ -1096,6 +1096,50 @@ Value *InstCombinerImpl::foldUsingDistributiveLaws(BinaryOperator &I) {
   return SimplifySelectsFeedingBinaryOp(I, LHS, RHS);
 }
 
+bool InstCombinerImpl::matchSymmetricPhiNodesPair(PHINode *LHS, PHINode *RHS) {
+
+  if (LHS->getNumOperands() != 2 || RHS->getNumOperands() != 2)
```

Comment:
```suggestion
  if (LHS->getNumIncomingValues() != 2 || RHS->getNumIncomingValues() != 2)
```
It would be clearer than `getNumOperands`.